src/omnisharp/extension.ts

```diff
@@ -152,8 +152,14 @@ export async function activate(context: vscode.ExtensionContext, packageJSON: an
             });
     }));
 
-    // read and store last solution or folder path
-    disposables.add(server.onBeforeServerStart(path => context.workspaceState.update('lastSolutionPathOrFolder', path)));
+    disposables.add(server.onBeforeServerStart(path => {
+        if (options.razorDevMode) {
```

Comment:
I couldn't find a better place to put this. If I try and dispatch a event before the server's start processes has begun the log message gets completely lost.